api/questions/urls.py

```diff
@@ -16,4 +16,6 @@
          QuestionEditViews.as_view(), name='delete_question'),
     path('<str:m_id>/questions/<str:id>/',
          QuestionEditViews.as_view(), name='edit_question'),
+    path('<str:m_id>/questions/<str:id>/question',
```

Comment:
This `/question` at the end is unnecessary @a-braham. The URL should be `GET /api/meetups/{meetupId}/questions/{questionId}`